docs/releases/4.0.txt

```diff
@@ -399,6 +399,8 @@ Miscellaneous
 * The undocumented ``django.contrib.admin.utils.lookup_needs_distinct()``
   function is renamed to ``lookup_spawns_duplicates()``.
 
+* The undocumented ``HttpRequest.get_raw_uri()`` method is removed.
```

Comment:
Maybe a mention of `build_absolute_uri` ?

Reply:
Added.